packages/create-amplify/src/logger.ts

```diff
@@ -79,6 +154,6 @@ export const argv = await yargs(process.argv.slice(2)).options({
 const minimumLogLevel =
   argv.debug || argv.verbose ? LogLevel.DEBUG : LogLevel.INFO;
```

Comment:
if we only have a debug log level for now, we should only have a debug arg

Reply:
Done in [2952c9a](https://github.com/aws-amplify/amplify-backend/pull/595/commits/2952c9a94a0c0ed95148bd5a241fa5a50af989ac)